vertical-pod-autoscaler/hack/vpa-process-yaml.sh

```diff
@@ -31,8 +31,8 @@ if [ $# -eq 0 ]; then
   exit 1
 fi
 
-DEFAULT_REGISTRY="k8s.gcr.io"
-DEFAULT_TAG="v0.0.1"
+DEFAULT_REGISTRY="gcr.io/google_containers/vpa-updater"
```

Comment:
Should be just "gcr.io/google_containers".

Reply:
Done.